travis-test.sh

```diff
@@ -2,14 +2,14 @@
 echo "Branch: $TRAVIS_BRANCH"
 make --no-print-directory testbin -C p
 ERRORED=$?
-if [[  "$ERRORED" == "1" ]]; then
+if [[ "$ERRORED" == "1" ]]; then
```

Comment:
what about using posix shell instead of bashisms?

Reply:
i can change it later.

Reply:
changed when regressions were moved.